src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -7081,6 +7081,68 @@ public class Methods2
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public void SharedProjectFieldAndPropertiesTreatedAsIdentical()
```

Comment:
@DustinCampbell The language doesn't really matter--this is really about testing the behavior of that comparer (and all the related tests are in C# :smile: ).